### ISSUE TITLE
Error when pro-populating dropdown element fields

### ISSUE THREAD

**Issue by bartrylant** [verbb/formie#1413]
### Describe the bug

I have an element field and have set a query parameter (playerid) to prepopulate the field. However, if I add e.g. `?playerid=827574` to the url (where 827574 is the id of the element I want to be selected in the dropdown), I get
```
foreach() argument must be of type array|object, string given
```
If I do `?playerid[]=827574`, the dropdown field is empty.

### Steps to reproduce

1. Set a query parameter as a prepopulate value on an element dropdown field

### Form settings

- Multi-page form: No
- Submission Method: Ajax 
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

4.4.8

### Plugin version

2.0.28

### Multi-site?

Yes

**Comment by engram-design** [comment]
Can't seem to replicate that, sorry. Can you post the full stack trace of the error with `devMode` enabled?

**Comment by engram-design** [comment]
Thanks for clarifying, looks to be due to using limit settings too. Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.0.28"`

**Comment by engram-design** [comment]
Fixed in [2.0.29](https://github.com/verbb/formie/releases/tag/2.0.29)
